### ISSUE TITLE
Remote Market Bug

### ISSUE THREAD

**Issue by MikeJeffers** [OpenPerpetuum/OP-Project#28]
Displaying Remote market on any item emits server error

```
[16:27:26] ERR 
Exception Found:
Type: System.NullReferenceException
Message: Object reference not set to an instance of an object.
Source: Perpetuum
Stacktrace:    at Perpetuum.Services.MarketEngine.MarketPriceCollector.GetBaseEid()
   at Perpetuum.Services.MarketEngine.MarketHandler.GetPriceCollectorByMarket(Market market)
   at Perpetuum.Services.MarketEngine.MarketHandler.TryGetDockingBaseEidForMarketEid(Int64 marketEid, Int64& baseEid)
   at Perpetuum.Services.MarketEngine.MarketOrder.GetBaseEid()
   at Perpetuum.Services.MarketEngine.MarketOrder.ToDictionary()
   at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   at Perpetuum.RequestHandlers.Markets.MarketItemsInRange.HandleRequest(IRequest request)
   at Perpetuum.Profiler.ExecutionTimeOf(Action action)
   at Perpetuum.Host.Requests.RequestHandlerProfiler`1.HandleRequest(T request)
   at Perpetuum.Services.Sessions.Session.OnDataReceived(ITcpConnection connection, Byte[] receivedData)
```

**Comment by Sellafield** [comment]
Please provide with the error message, not just stack trace

**Comment by MikeJeffers** [comment]
Woops sorry, sloppy copy-paste.  Updated issue text

**Comment by MikeJeffers** [comment]
Another clue: Error only occurs on items that are typically seeded by NPC orders.
T1 mods, ammo, etc.
Other items do not yield error.

Suspect that dbo.vendors/vendorpresets that are present and associated to market EID ('vendors' and 'markets' are entities in dbo.entities.  vendors are children of markets 1:1)

Anyone want to write a SQL script that removes vendors on disabled zones?

**Comment by MikeJeffers** [comment]
@OpenPerpetuum/db-team this is a database fix.  please look into

**Comment by MikeJeffers** [comment]
Closed by OpenPerpetuum/PerpetuumServer@ce1cce08fc1bd86b3e99b62ea0f603ad6aaf340
